docs/components/base/list.tsx

```diff
@@ -69,19 +69,38 @@ const STYLES_LIST_ITEM = css`
 `;
 
 const STYLE_RETURN_LIST = css`
-  list-style-type: '⇒';
+  list-style-type: '⮑';
   padding-left: 0.5rem;
+  margin-left: 0.25rem;
+
+  ::marker {
+    color: ${theme.text.secondary};
```

Comment:
```suggestion
    color: ${theme.icon.secondary};
```
we might try even lighter